probe-rs/targets/esp32.yaml

```diff
@@ -71,6 +71,7 @@ flash_algorithms:
   pc_erase_sector: 0xa8
   pc_erase_all: 0xb8
   data_section_offset: 0x400902f8
+  stack_overflow_check: false
```

Comment:
ESP flashing algos completely ignore probe-rs's intentions toward stacks, attempting the overflow check would just cause problems here.